content/en/users/Rucio/Rucio-Admin/_index.md

```diff
@@ -0,0 +1,353 @@
+---
+title: "Rucio Admin"
+type: docs
+linkTitle: "Rucio Admins"
+weight: 100
+description: >-
+     Information for Rucio admins to better understand and perform actions for their VO.
+---
+
+
+## Creating Accounts and identities
+
+New users within your VO, the account will need to be created before adding identities using the same methods as above
```

Comment:
i guess you are referring to the steps for asking for a new account as in the getting started right? so it needs to be linked

Reply:
Rewrote this section to make more sense and expand on hierarchy of Rucio admins